src/mixer/basetrackplayer.cpp

```diff
@@ -134,8 +136,41 @@ BaseTrackPlayerImpl::BaseTrackPlayerImpl(QObject* pParent,
             SLOT(slotWaveformZoomSetDefault(double)));
 
     m_pPreGain = std::make_unique<ControlProxy>(group, "pregain", this);
-    // BPM of the current song
 
+    m_pShiftCuesEarlier = std::make_unique<ControlPushButton>(
+            ConfigKey(group, "shift_cues_earlier"));
+    connect(m_pShiftCuesEarlier.get(),
+            &ControlObject::valueChanged,
+            this,
+            [this](double value) { slotShiftCuesButton(value, -1 * kShiftCuesOffsetMillis); });
+    m_pShiftCuesLater = std::make_unique<ControlPushButton>(
+            ConfigKey(group, "shift_cues_later"));
+    connect(m_pShiftCuesLater.get(),
+            &ControlObject::valueChanged,
+            this,
+            [this](double value) { slotShiftCuesButton(value, kShiftCuesOffsetMillis); });
+    m_pShiftCuesEarlierSmall = std::make_unique<ControlPushButton>(
+            ConfigKey(group, "shift_cues_earlier_small"));
+    connect(m_pShiftCuesEarlierSmall.get(),
+            &ControlObject::valueChanged,
+            this,
+            [this](double value) {
+                slotShiftCuesButton(value, -1 * kShiftCuesOffsetSmallMillis);
```

Comment:
Why not simply `-kShiftCuesOffsetSmallMillis`? Also above.

Reply:
I didn't realize this shorthand was allowed in C++.